src/webpack.config.ts

```diff
@@ -0,0 +1,64 @@
+export function createWebpackConfiguration(config: any): any {
```

Comment:
nit: can we rename this file to `webpackConfig.ts`? This is not really a config file anymore

Reply:
yup